src/app.jsx

```diff
@@ -711,22 +711,28 @@ class Application extends React.Component {
             imageContainerList = null;
 
         let startService = "";
-        const action = (<>
-            <AlertActionLink variant='secondary' onClick={this.startService}>{_("Start")}</AlertActionLink>
-            <AlertActionCloseButton onClose={() => this.setState({ showStartService: false })} />
-        </>);
+        const action = (
+            <>
+                <AlertActionLink variant='secondary' onClick={this.startService}>{_("Start")}</AlertActionLink>
+                <AlertActionCloseButton onClose={() => this.setState({ showStartService: false })} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1217-20230227-180700-70c39418-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)